src/video/SDL_sysvideo.h

```diff
@@ -345,6 +345,7 @@ struct SDL_VideoDevice
     Uint32 next_object_id;
     char *clipboard_text;
     SDL_bool setting_display_mode;
+    SDL_bool no_display_mode_switching;
```

Comment:
Where is this variable set to FALSE?
Also, please name it in a positive way for easier reading, e.g. allow_display_mode_switching or enable_display_mode_switching